extension/src/views/defaultProject/DefaultProjectProvider.ts

```diff
@@ -128,7 +128,9 @@ export class DefaultProjectProvider {
         command: 'vscode.open',
         arguments: [
           buildFileUri,
-          <vscode.TextDocumentShowOptions>{ selection: dependencyItem.range },
+          <vscode.TextDocumentShowOptions>{
```

Comment:
so here you prefer `<type>` casting method instead of `as type`?

Reply:
This behavior is affected by [`@typescript-eslint/consistent-type-assertions`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-assertions.md) and I just find it's not included in the recommended configuration. So the current behavior is not to change the existing type assertions.

Since `<type>` is not supported in jsx files (although we may not include them in this project), I'll add a rule to enforce the code base to use `as type`. See the reference: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-assertions.md#assertionstyle

Reply:
I'm ok with both, just make sure you use consistent lint rules.